### ISSUE TITLE
Update SYN_RF_R_REG (Synthesizer Register A) to allow changing the RX frequency in steps of less than 2 MHz.

### ISSUE THREAD

**Issue by sasodoma** [sasodoma/fpv-receiver#1]
I am opening this issue myself for the purpose of documenting what I have discovered while attempting to do what the title says.

**TL;DR: It can be done, if you first write to `Register B` (setting the `N` and `A` counters), wait 27 milliseconds or more, and then write to `Register A`.**

The RTC6715 calculates the receiving frequency using the following formula: `F_RF [MHz] = 479 - 2 * (N * 32 + A) * (8 / R)`. The `N`, `A`, and `R` values are set in the registers `0x00: Synthesizer Register A` (`R` is bits 14-0) and `0x01: Synthesizer Register B` (`N` is bits 19-7 and `A` is bits 6-0). The default value for `R` is `8`, for `N` it's `84` and for `A` it's `5`. This gives a frequency of 5865 MHz. The frequency is normally set by changing `N` and `A`, but since `2 * (8 / R) = 2`, the frequency can only be changed in steps of 2. Together with the IF frequency of 479 MHz this means that only the odd frequencies can be tuned to.

On the paper, the solution is simple. Changing `R` to 16 gives `2 * (8 / R) = 1` which means frequency can be adjusted in steps of 1 MHz. In practice there are two major problems with this solution and both are related to strange behavior of the `Register B`.
Firstly, writing to the `Register B` seems to always reset `Register A` to its default value of `8`. This means that `R` cannot be set as part of the initalization sequence but needs to be set after every frequency change. Attempting to do this, however, reveals an even stranger bug. After writing to `Register B`, the SPI interface of the chip seems to glitch out. For about 27 ms after the write, any subsequent writes to any other register will not only update the other register **but also update `Register B` with the same value**. Then, since `Register B` gets updated, `Register A` also gets reset to the default value. So the result of writing first to `Register B` and then to `Register A` is `Register B` having the value we wanted to write to `Register A` and `Register A` having the default value of `8`. By some brute force testing I have determined that waiting at least 27 ms after writing to `Register B` will result in both registers being set correctly and the receiver will tune to the proper frequency.

This is a school project, where one of the requirements is using a simple cooperative RTOS. Each time slice is only 5 ms, so I don't have the time to wait inside a time slice. The waiting could be implemented using flags to set registers in different iterations of the time slice, but I have decided that 2 MHz steps are acceptable for me.
